corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -47,8 +54,10 @@ hqDefine("geospatial/js/case_grouping_map",[
 
             const casesToExport = _.map(self.casesToExport(), function (caseItem) {
                 const coordinates = (caseItem.coordinates) ? `${caseItem.coordinates.lng} ${caseItem.coordinates.lat}` : "";
+                const groupCoordinates = (caseItem.groupCoordinates) ? `${caseItem.groupCoordinates.lng} ${caseItem.groupCoordinates.lat}` : "";
```

Comment:
Nit: Not blocking, but I wonder if we can refactor this to be a function inside the `caseModel` class? Perhaps something like `toJson()` which will return the JSON object below, and then here we can simply call `return caseItem.toJson()` for each case object.